src/ui/components/Comments/TranscriptComments/CommentEditor/TipTapEditor.tsx

```diff
@@ -90,7 +94,7 @@ const TipTapEditor = ({
         }),
       ],
       editorProps: { attributes: { class: "focus:outline-none" } },
-      content: tryToParse(content),
+      content: typeof content === "string" ? tryToParse(content) : content,
```

Comment:
this feels like something that might live in `tryToParse`

`content: tryToParse(content)` could just return if it's an object